EasyCommands/CommandParsers/ParameterParsers.cs

```diff
@@ -364,6 +368,22 @@ partial class Program {
             else AddWords(words, new PropertyCommandParameter(property));
         }
 
+        void AddAmbigiuousControlWords(String[] words, ControlFunction function)
+        {
+            AddAmbiguousWords(words, new CommandReferenceParameter(new ControlCommand { controlFunction = function }));
+        }
+
+        void AddAmbiguousPropertyWords(String[] words, Property property, bool nonNegative = true) {
+            if (!nonNegative) AddAmbiguousWords(words, new PropertyCommandParameter(property), new BooleanCommandParameter(false));
+            else AddAmbiguousWords(words, new PropertyCommandParameter(property));
+        }
+
```

Comment:
I think these functions may not be necessary yet?  If we just use AddAmbiguousWords, does that save characters?

Reply:
`AddAmbiguousControlWords(...)` is atleast right now completly unnecessary.
`AddAmbiguousPropertyWords(...)` only purpose right now is to makes the definitions of the tokens look the same.

Reply:
let's condense for now to save chars and add the utility functions once it's less chars to use the utility vs not having it.  This is the pattern I try to follow elsewhere so that we aren't constantly hitting the character limit and having to refactor stuff to stay below 100000.